src/utils/sentry.ts

```diff
@@ -20,7 +20,7 @@ export const addRequestScopeToSentry = (
 ): void => {
   const headers = Object.entries(request.headers).reduce(
     (acc, [key, value]) =>
-      key.toLowerCase() !== ("cookie" || "authorization")
+      key.toLowerCase() !== "cookie" && key.toLowerCase() !== "authorization"
```

Comment:
```suggestion
      !["cookie", "authorization"].includes(key.toLowerCase())
```

Reply:
Damn that's beautiful, I had to make the change ahah. I was wondering, should I put this in a function and test it quickly? Looks important.